plugins/modules/aci_l3out_hsrp_interface_profile.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Shreyas Srish (@shrsr) <ssrish@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l3out_hsrp_interface_profile
+short_description: Manages Layer 3 Outside (L3Out) HSRP interface profile (hsrp:IfP)
+description:
+- Manages L3Out HSRP interface profile on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  l3out:
+    description:
+    - Name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+  node_profile:
+    description:
+    - Name of the node profile.
+    type: str
+    aliases: [ node_profile_name, logical_node ]
+  interface_profile:
+    description:
+    - Name of an existing interface profile.
+    type: str
+    aliases: [ name, interface_profile_name, logical_interface ]
+  hsrp_policy:
+    description:
+    - Name of an existing hsrp interface policy.
```

Comment:
Do we need to add this attribute even though it's not relevant for the filtering of this object? And is it configurable in the UI?

Reply:
hsrp_policy is an optional attribute and a child class to this object, so it is part of this module. It is configurable in the UI